src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -57,6 +60,20 @@ internal bool NeedsGeneratedIsReadOnlyAttribute
             }
         }
 
+        /// <summary>
+        /// Returns a value indicating whether this builder has a symbol that needs IsReadOnlyAttribute to be generated during emit phase.
```

Comment:
IsByRef

---
In reply to: [118795013](https://github.com/dotnet/roslyn/pull/19795#discussion_r118795013) [](ancestors = 118795013)